api/auth.go

```diff
@@ -58,16 +58,6 @@ func NewAuth(key *ecdsa.PrivateKey, customers customers.Repository, passwd strin
 	}
 }
 
-// CreateCustomerJWT creates a JWT that only stores the customer ID. This is required for the IRMA flow.
```

Comment:
I don't think you can safely remove this code without fixing the IRMA login flow.